src/Models/UserInput.cs

```diff
@@ -33,6 +34,7 @@ bool assessSqlServicesSeparately
 
             AzureMigrateSourceAppliances = azureMigrateSourceAppliances;
             WorkflowObj = new Workflow(isExpressWorkflow, module);
+            IsAvsProposal = isAvsProposal;

```

Comment:
Can this be an ENUM.

Instead of setting a value in boolean format

Please create an ENUM
Enum ProposalTypes {
Comprehensive,
AVS,
}

And then pass that ENUM type and check the same.

Reply:
I followed similar format to: bool isExpressWorkflow